app/src/main/java/com/okta/idx/android/directauth/sdk/FormAction.kt

```diff
@@ -82,26 +82,27 @@ data class FormAction internal constructor(
 
             return when (response.authenticationStatus) {
                 AuthenticationStatus.AWAITING_PASSWORD_RESET -> {
-                    registerPasswordForm(response)
+                    registerPasswordForm(response, "Reset Password")
                 }
                 AuthenticationStatus.PASSWORD_EXPIRED -> {
-                    registerPasswordForm(response)
+                    registerPasswordForm(response, "Password Expired")
                 }
                 AuthenticationStatus.AWAITING_AUTHENTICATOR_SELECTION -> {
-                    selectAuthenticatorForm(response)
+                    selectAuthenticatorForm(response, "Select Authenticator")
                 }
                 AuthenticationStatus.AWAITING_AUTHENTICATOR_VERIFICATION -> {
```

Comment:
Does `Verify Authenticator` not need a title too?

Reply:
Yeah, I just hard coded it though. https://github.com/okta/okta-idx-android/pull/23/files#diff-a7d27b7e2ff2ee67956be28e4997655bd6e436ba48f4eb1d0c4cbfa4305ed590R16

No matter the state it's coming from, it's always displaying as "Verify Code"